package-res/reportviewer/reportviewer.js

```diff
@@ -1252,6 +1425,60 @@ define([ 'common-ui/util/util', 'common-ui/util/timeutil', 'common-ui/util/forma
         });
       },
 
+      _buildParameter: function (pathArray, id) {
+        var path;
+        for (var i = 0; i < pathArray.length; i++) {
+          if (pathArray[i].indexOf(".prpt") !== -1) {
+            path = decodeURIComponent(pathArray[i]).replace(/:/g, "/");
+          }
+        }
+        return {
+          solutionPath: (path == null ? "/" : path ),
+          jobId: id,
+          recalculateFinished: true == this.reportPrompt._isReportHtmlPagebleOutputFormat
+        };
+      },
+
+      _getSuccessSchedulingDlg: function () {
+
+        var me = this;
+        var successDlg = registry.byId('successScheduleScreen');
+        successDlg.setTitle(_Messages.getString('SuccessScheduleTitle'));
+        successDlg.setText(_Messages.getString('SuccessScheduleText', '<b><i>' + me._locationOutputPath +'</i></b>'));
+        successDlg.setOkBtnText(_Messages.getString('OK'));
+
+        var successScheduleDialogCallbacks = [
+          function hide() {
+            me._forceHideGlassPane();
+            successDlg.hide();
+          }.bind(me)
+        ];
+
+        successDlg.callbacks = successScheduleDialogCallbacks;
+        return  successDlg;
+      },
+      
+      _forceHideGlassPane: function (){
+        $("#glasspane").css("background", "transparent");
+        $("#glasspane").css("display", "none");
+      },
+
+      _forceShowGlassPane: function (){
+        $("#glasspane").css("background", "");
+        $("#glasspane").css("display", "block");
+      },
+      
+      _removeLocationPromptHandlers: function(){
+        if (window.top.mantle_removeHandler) {
+          if(this._locationPromptCancelHandlerRegistration) {
+            window.top.mantle_removeHandler(this._locationPromptCancelHandlerRegistration);
+          }
+          if(this._locationPromptOkHandlerRegistration) {
```

Comment:
Where is the removal of the "finish" handler? Three go in, two come out, and one parties on to create a memory leak :)